android/app/src/main/res/values/styles.xml

```diff
@@ -4,5 +4,9 @@
     <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
         <!-- Customize your theme here. -->
         <item name="android:textColor">#000000</item>
+
+        <item name="colorPrimary">#2B4380</item>
+        <item name="colorPrimaryDark">#26374A</item>
+        <item name="colorControlActivated">#26374A</item>
```

Comment:
This will help with the cursor color for Android.

Reply:
iOS is ready done here https://github.com/cds-snc/covid-shield-mobile/blob/master/ios/CovidShield/AppDelegate.m#L87